### PR TITLE
docs: additional policy, job run and Deny default sec group

### PR DESCRIPTION
**Pull request by paragbhingre**
Adding docs for additional policy, job run and Deny default sec group features

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v120.en.md [hunk 1]
```diff
@@ -18,7 +18,12 @@ Copilot v1.20 brings several new features and improvements:
 
 * **Environment manifests**: You can now create and update environments with a [manifest file](../docs/manifest/environment.en.md) bringing all the benefits of infrastructure as code to environments. 
    [See detailed walkthrough](#environment-manifest) for how to migrate your existing environments.
-
+* **Additional policy to build role**: You can now specify additional policy for CodeBuild Build Project Role through pipeline manifest field `additional_policy`.
```
Comment (dannyrandall) [inline site/content/blogs/release-v120.en.md]:
```suggestion
* **Additional policy to build role**: You can now specify an additional policy for the CodeBuild Build Project Role through the pipeline manifest field `additional_policy`.
```

File: site/content/blogs/release-v120.en.md [hunk 2]
```diff
@@ -18,7 +18,12 @@ Copilot v1.20 brings several new features and improvements:
...
+  [See detailed walkthrough](../docs/manifest/pipeline.en.md) for how to specify an additional policy document to add to the build project role. [(#3709)](https://github.com/aws/copilot-cli/pull/3709)
+* **`job run`**: Command now can be used to execute scheduled jobs.
+  [(#3692)](https://github.com/aws/copilot-cli/pull/3692)
+* **Deny default sec group**: This change adds option to deny EnvironmentSecurityGroup which is applied by default on copilot service
```
Comment (dannyrandall) [inline site/content/blogs/release-v120.en.md]:
```suggestion
* **Deny default security group**: This change adds the option to deny other services in an environment access to your service.
```
maybe this sounds better? (tbh i don't know the feature super well so if i'm wrong pls ignore me)

File: site/content/blogs/release-v120.en.md [hunk 3]
```diff
@@ -18,7 +18,12 @@ Copilot v1.20 brings several new features and improvements:
 
 * **Environment manifests**: You can now create and update environments with a [manifest file](../docs/manifest/environment.en.md) bringing all the benefits of infrastructure as code to environments. 
    [See detailed walkthrough](#environment-manifest) for how to migrate your existing environments.
-
+* **Additional policy to build role**: You can now specify additional policy for CodeBuild Build Project Role through pipeline manifest field `additional_policy`.
+  [See detailed walkthrough](../docs/manifest/pipeline.en.md) for how to specify an additional policy document to add to the build project role. [(#3709)](https://github.com/aws/copilot-cli/pull/3709)
+* **`job run`**: Command now can be used to execute scheduled jobs.
```
Comment (efekarakus) [inline site/content/blogs/release-v120.en.md]:
```suggestion
* **Invoke a scheduled job**: You can now execute an existing scheduled job ad hoc using the new `copilot job run` command.
```

File: site/content/blogs/release-v120.en.md (hunk 2 above)
Comment (efekarakus) [reply site/content/blogs/release-v120.en.md]: Perhaps the following?
```suggestion
* **Deny default sec group**: Add an option `deny_default` to `security_groups` in service manifests to remove the EnvironmentSecurityGroup ingress that is applied by default.
```
It'd be awesome if we can link to the manifest fields here

Comment (paragbhingre) [reply site/content/blogs/release-v120.en.md]: I don't see any changes in the docs/manifest to refer these flags to the manifest fields on our docs.

**Review by efekarakus — APPROVED** [review]

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**